health_check.py

```
@@ -0,0 +1,41 @@
+class BackgroundLoopHealthCheck(HealthCheck):
```

Flake8 (7.3.0)

[error] undefined name 'HealthCheck'

(F821)